src/gtk/artgtk.cpp

```diff
@@ -122,27 +133,23 @@ const wxString wxId2Gtk[] = {
     ART(wxART_FIND_AND_REPLACE, GTK_STOCK_FIND_AND_REPLACE, "edit-find-replace")
     ART(wxART_FULL_SCREEN,      GTK_STOCK_FULLSCREEN, "view-fullscreen")
     ART(wxART_EDIT,             "accessories-text-editor", "accessories-text-editor")
-};
-
-#undef ART
+  };
 
-wxString wxArtIDToStock(const wxArtID& id)
-{
     // allow passing GTK+ stock IDs to wxArtProvider -- if a recognized wx
     // ID wasn't found, pass it to GTK+ in the hope it is a GTK+ or theme
     // icon name:
     wxString ret(id);
```

Comment:
This is unused now and should be removed (or the change to the last line of this function should be undone and `break` in the loop be restored).